src/component/pages/main-page/main-page.tsx

```diff
@@ -15,7 +16,9 @@ export const MainPage: FC = () => {
     <ThemeProvider theme={theme}>
       <Grid className="container" container>
         <Grid item>
-          <Navbar categories={category} />
+          <CartProvider>
```

Comment:
Is CartProvider applied only to the Navbar?

Reply:
Cart Provider is for the entire project only for testing purposes, it is used here.